<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1380</width>
    <height>750</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1380</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1380</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Alisa Corporation :: Payment System</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(60, 60, 60);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>10</y>
      <width>290</width>
      <height>660</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(24,24,24);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>271</width>
       <height>136</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_misc_id">
        <property name="styleSheet">
         <string notr="true">color: rgb(177, 0, 0);
font: 900;</string>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="input_misc_id">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(60, 60, 60);
color: #666;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_misc_txid">
        <property name="styleSheet">
         <string notr="true">color: rgb(177, 0, 0);
font: 900;</string>
        </property>
        <property name="text">
         <string>Tx ID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="input_misc_txid">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(60, 60, 60);
color: #666;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_misc_created">
        <property name="styleSheet">
         <string notr="true">color: rgb(177, 0, 0);
font: 900;</string>
        </property>
        <property name="text">
         <string>Created</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="input_misc_created">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(60, 60, 60);
color: #666;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_misc_updated">
        <property name="styleSheet">
         <string notr="true">color: rgb(177, 0, 0);
font: 900;</string>
        </property>
        <property name="text">
         <string>Updated</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="input_misc_updated">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(60, 60, 60);
color: #666;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_misc_delayed">
        <property name="styleSheet">
         <string notr="true">color: rgb(177, 0, 0);
font: 900;</string>
        </property>
        <property name="text">
         <string>Delayed</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="input_misc_delayed">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(60, 60, 60);
color: #666;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="section_label_misc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(96, 96, 96);
font: 900;</string>
     </property>
     <property name="text">
      <string>Misc</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="section_label_payment">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(96, 96, 96);
font: 900;</string>
     </property>
     <property name="text">
      <string>Payment</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>271</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_payment_sale_id">
        <property name="styleSheet">
         <string notr="true">color: rgb(177, 0, 0);
font: 900;</string>
        </property>
        <property name="text">
         <string>Sale ID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="input_payment_sale_id">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(60, 60, 60);
color: #666;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_payment_ex_txid">
        <property name="styleSheet">
         <string notr="true">color: rgb(177, 0, 0);
font: 900;</string>
        </property>
        <property name="text">
         <string>ex. Tx ID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="input_payment_ex_txid">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(60, 60, 60);
color: #666;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_payment_system">
        <property name="styleSheet">
         <string notr="true">color: rgb(177, 0, 0);
font: 900;</string>
        </property>
        <property name="text">
         <string>System</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="input_payment_system">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(60, 60, 60);
color: #666;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_payment_method">
        <property name="styleSheet">
         <string notr="true">color: rgb(177, 0, 0);
font: 900;</string>
        </property>
        <property name="text">
         <string>Method</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="input_payment_method">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(60, 60, 60);
color: #666;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_payment_status">
        <property name="styleSheet">
         <string notr="true">color: rgb(177, 0, 0);
font: 900;</string>
        </property>
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="input_payment_status">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(60, 60, 60);
color: #666;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="button_download_invoice">
        <property name="styleSheet">
         <string notr="true">color: rgb(177, 0, 0);
font: 900;</string>
        </property>
        <property name="text">
         <string>Download Invoice</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>271</width>
       <height>108</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_player_obj_id">
        <property name="styleSheet">
         <string notr="true">color: rgb(177, 0, 0);
font: 900;</string>
        </property>
        <property name="text">
         <string>Obj ID    </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="input_player_obj_id">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(60, 60, 60);
color: #666;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_player_name">
        <property name="styleSheet">
         <string notr="true">color: rgb(177, 0, 0);
font: 900;</string>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="input_player_name">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(60, 60, 60);
color: #666;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_player_email">
        <property name="styleSheet">
         <string notr="true">color: rgb(177, 0, 0);
font: 900;</string>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="input_player_email">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(60, 60, 60);
color: #666;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_player_server">
        <property name="styleSheet">
         <string notr="true">color: rgb(177, 0, 0);
font: 900;</string>
        </property>
        <property name="text">
         <string>Server</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="input_player_server">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(60, 60, 60);
color: #666;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>580</y>
       <width>271</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_item_id">
        <property name="styleSheet">
         <string notr="true">color: rgb(177, 0, 0);
font: 900;</string>
        </property>
        <property name="text">
         <string>Item ID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="input_item_id">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(60, 60, 60);
color: #666;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_item_quantity">
        <property name="styleSheet">
         <string notr="true">color: rgb(177, 0, 0);
font: 900;</string>
        </property>
        <property name="text">
         <string>Quantity</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="input_item_quantity">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(60, 60, 60);
color: #666;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="section_label_item">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>550</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(96, 96, 96);
font: 900;</string>
     </property>
     <property name="text">
      <string>Item</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="section_label_player">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(96, 96, 96);
font: 900;</string>
     </property>
     <property name="text">
      <string>Player</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1060</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(24,24,24);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLineEdit" name="input_search">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>20</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(60, 60, 60);
color: rgb(98, 98, 98);</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_search">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>20</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>false</italic>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(177, 0, 0);
font: 900;</string>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_dev_company">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(177, 0, 0);</string>
     </property>
     <property name="text">
      <string>Alisa Corporation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_dev_email">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(96, 96, 96);</string>
     </property>
     <property name="text">
      <string>alisa.corporation@protonmail.com</string>
     </property>
    </widget>
    <widget class="QLabel" name="logo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Pictures/alisa_corporation/logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>1061</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(24,24,24);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1061</width>
       <height>581</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableView" name="payment_table">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>953</x>
      <y>673</y>
      <width>174</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="cfg_font_color">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="theme_radio_red">
       <property name="styleSheet">
        <string notr="true">color: rgb(177, 0, 0);
font: 900;</string>
       </property>
       <property name="text">
        <string>Red</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="theme_radio_blue">
       <property name="styleSheet">
        <string notr="true">color: rgb(28, 113, 216);
font: 900;</string>
       </property>
       <property name="text">
        <string>Blue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="theme_radio_green">
       <property name="styleSheet">
        <string notr="true">color: rgb(38, 162, 105);
font: 900;</string>
       </property>
       <property name="text">
        <string>Green</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1126</x>
      <y>673</y>
      <width>128</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="cfg_font_weight">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="theme_font_weight_bold">
       <property name="styleSheet">
        <string notr="true">color: rgb(154, 153, 150);
font: 900;</string>
       </property>
       <property name="text">
        <string>Bold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="theme_font_weight_normal">
       <property name="styleSheet">
        <string notr="true">color: rgb(154, 153, 150);</string>
       </property>
       <property name="text">
        <string>Normal</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1258</x>
      <y>673</y>
      <width>119</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="cfg_font_color_2">
     <item>
      <widget class="QRadioButton" name="theme_font_color_white">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>white</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="theme_font_color_gray">
       <property name="styleSheet">
        <string notr="true">color: rgb(154, 153, 150);</string>
       </property>
       <property name="text">
        <string>gray</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>frame_3</zorder>
   <zorder>frame</zorder>
   <zorder>frame_2</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1380</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy_selection"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(24,24,24);
color: rgb(96,96,96);</string>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionCopy_selection">
   <property name="text">
    <string>Copy selection</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
